playbooks/Installation/installation.adoc

```diff
@@ -33,41 +40,82 @@ Kubernetes, and therefore OpenShift has a logical limit to the number of nodes t
 
 Docker's data directories live in `/var/lib/docker`, so the disk space should be allocated to that mount point. As with the OpenShift DataStore, it is highly recommended to give Docker a dedicated partition or volume so as to protect it from over-crowding by logging or other administrative services.
 
-===== Backups, Shared Storage & Replication
+NOTE: This section is referring to local storage used be docker to run containers on a host. This *DOES NOT* refer to running a docker registry.
+
+====== Backups, Shared Storage & Replication
 
 Nodes are considered stateless in a Kubernetes architecture, meaning that in the event of a loss of a Node, the Scheduler will immediately replace lost containers elsewhere in the environment. Therefore, no specialized sharing or replication is required at the node level to account for loss.
 
-===== Configuration Considerations
+====== Configuration Considerations
 
 The `docker` package ships with a command for setting up docker storage, `docker-storage-setup`. This script expects us to create a volume group for it in order to point it to the storage we've allocated.
 
-==== Example Storage Configuration
+===== Logs
+
+Blah.
+
+==== Storage Configuration Implementation
+
+===== Local Disk Config
+
+Create the following partitions (disk partitions or LVMs) on the system disk image:
+
+* Boot partition (`/boot`)
+* OS Root (`/`)
+* etcd storage (`/var/lib/openshift` or `/var/lib/etcd`)
+* Log storage (`/var/log`) - Optional, but recommended
+
+===== Docker Storage Config
+
+The OpenShift docs discuss link:https://docs.openshift.com/enterprise/3.0/install_config/install/prerequisites.html#configuring-docker-storage[three different options for configuring docker storage]. We consider options A and B to be "production ready".
+
+For option A: Attach an additional volume or block device to each node for Docker storage (20-50 GB)
+
+For option B: Leave unallocated space (20 - 50 GB) on your local disk. From the unallocated space, create the following volume group:
+
+* `docker-vg`
+
+===== Example: Configuring Host Storage with Docker Storage Option A
 
 To account for the above DataStore & Container storage needs, we are going to attach a single 25 GB volume and partition it to accommodate our needs. Here is a sample vdb layout for the disk:
 
 NOTE: The following must be done on ALL NODES, including the Master. However, if you are not going to be using the master as a node, then a minimal volume size can be used, as no containers will be hosted. Also, for NODES that are NOT MASTERS, the second DataStore partition is not needed.
 
-```bash
+[source,bash]
+----
 cat << EOF > /tmp/ose_vdb_layout.txt
```

Comment:
Instead of using "sfdisk", I'd provide an example like the following:

```
NOTE: actual numeric values will be different for each running system, but the steps outlined should be the same for all

>> fdisk /dev/vdb

Command (m for help): n <enter>
Partition type:
   p   primary (0 primary, 0 extended, 4 free)
   e   extended
Select (default p): p <enter>
Partition number (1-4, default 1):  <enter>
First sector (2048-16777215, default 2048): <enter>
Using default value 2048
Last sector, +sectors or +size{K,M,G} (2048-16777215, default 16777215): +20G <enter>
Partition 1 of type Linux and of size 20 GiB is set

Command (m for help): n <enter>
Partition type:
   p   primary (1 primary, 0 extended, 3 free)
   e   extended
Select (default p): p <enter>
Partition number (2-4, default 2): <enter>
First sector (12584960-16777215, default 12584960): <enter>
Using default value 12584960
Last sector, +sectors or +size{K,M,G} (12584960-16777215, default 16777215): +2G <enter>
Partition 2 of type Linux and of size 2 GiB is set

Command (m for help): t <enter>
Partition number (1,2, default 2): 1 <enter>
Hex code (type L to list all codes): 8e <enter>
Changed type of partition 'Linux' to 'Linux LVM'

Command (m for help): t <enter>
Partition number (1,2, default 2): 2 <enter>
Hex code (type L to list all codes): 8e <enter>
Changed type of partition 'Linux' to 'Linux LVM'

Command (m for help): w <enter>
The partition table has been altered!

Calling ioctl() to re-read partition table.
Syncing disks.
```